empyrical/perf_attrib.py

```diff
@@ -0,0 +1,141 @@
+import pandas as pd
+
+
+def perf_attrib(returns,
+                positions,
+                factor_returns,
+                factor_loadings):
+    """
+    Attributes the performance of a returns stream to a set of risk factors.
+
+    Performance attribution determines how much each risk factor, e.g.,
+    momentum, the technology sector, etc., contributed to total returns, as
+    well as the daily exposure to each of the risk factors. The returns that
+    can be attributed to one of the given risk factors are the
+    `common_returns`, and the returns that _cannot_ be attributed to a risk
+    factor are the `specific_returns`, or the alpha. The common_returns and
```

Comment:
My instinct would probably be to leave out the reference to alpha here.  I get the impression that that's kind of a loaded term, and that we use "specific vs. common returns" in part to avoid confusion around what different people expect/mean by "alpha".

Reply:
Yup makes sense. My thought in including it was that people might look for "alpha" when poking around empyrical, and might gloss over specific/common returns. But I can remove it if it obscures rather than clarifies what perf attrib is meant for

Reply:
cc @jzhang18 for thoughts on this.

Reply:
changed "or the alpha" to be "or the alpha returns" on @jzhang18's suggestion

Reply:
discussed a little more, the reference to alpha is removed